tests/test_mutualauth.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import unittest
+
+import falcon
+from falcon_mutualauth.mutualauth import Authorize
+
+
+class FakeRequest(object):
+
+    def __init__(self, role):
+        self.__role = role
```

Comment:
Actually, private vars use double underscores. 

On that note, I am guessing the convention on the team is to stick to private vars. However, I've personally gotten into a habit of only using protected variables as private ones are annoying to access if you ever need to. Ironically this ended up being a rule in Effective Python (https://hacktec.gitbooks.io/effective-python/content/en/Chapter3/item27.html) so I'd be in favor of using protected here instead.